packages/frontend/package.json

```diff
@@ -12,6 +12,7 @@
     "validate": "yarn format:check && yarn lint"
   },
   "dependencies": {
+    "daisyui": "^4.0.4",
```

Comment:
Why is `daisyui` no longer in dev deps?

Reply:
Ah, good point. First I thought it's an issue with daisy. Moved back to dev deps.